### ISSUE TITLE
`genSources` task no longer depends on `genSourcesWithQuiltflower` task

### ISSUE THREAD

**Issue by Kneelawk** [QuiltMC/quilt-loom#33]
# The Issue
It seems that the `genSources` task no longer starts the `genSourcesWithQuiltflower` task by default, instead starting the `genSourcesWithCfr` task.

# Workaround
It is possible to work around this issue via:

Groovy:
```gradle
afterEvaluate {
    tasks.named("genSources") {
        dependsOn = ["genSourcesWithQuiltflower"]
    }
}
```

Kotlin:
```kotlin
afterEvaluate {
    tasks.named("genSources") {
        setDependsOn(listOf("genSourcesWithQuiltflower"))
    }
}
```

**Comment by Kneelawk** [comment]
Something I forgot to mention earlier is that this is happening in a split-env project.

**Comment by violet-sh** [comment]
Should be fixed by f0e73ca
